app/config/config.yml

```diff
@@ -173,6 +173,13 @@ surfnet_stepup:
         originator: %sms_originator%
         otp_expiry_interval: %sms_otp_expiry_interval%
         maximum_otp_requests: %sms_maximum_otp_requests%
+    locale_cookie:
+        name: stepup_locale
+        domain: %locale_cookie_domain%
+        expire: 0
+        path: /
+        secure: true
+        httpOnly: false
```

Comment:
There are no secrets in it, seems useful to let client side code access the value.